rmgpy/molecule/adjlist.py

```diff
@@ -772,7 +836,7 @@ def from_adjacency_list(adjlist, group=False, saturate_h=False):
             Saturator.saturate(atoms)
 
     # Consistency checks
-    if not group:
+    if not group and check_consistency:
```

Comment:
Why adding the option to skip consistency check?

Reply:
If you allow consistency checking for a molecule with reaction bonds, it will get upset because it has bonds RMG doesn't expect it to have, it naturally won't match RMG atomtypes properly.

Reply:
I might merge this PR anyway, but should we instead (or as well) make the consistency checkers cope with reaction bonds? Currently you could create an adjacancy list with reaction bonds AND other inconsistencies, and not notice the other inconsistencies.

Reply:
This is true, it would be good to at least consistency check parts of the Molecule that don't have reaction bonds.

Reply:
I'll let you open a new issue (or mental note) to fix this :)